include/mgmt/rpc/jsonrpc/JsonRPCManager.h

```diff
@@ -244,8 +244,8 @@ class JsonRPCManager
       InternalHandler(const RPCRegistryInfo *info, TSRPCHandlerOptions const &opt) : _regInfo(info), _options(opt) {}
       /// Sets the handler.
       template <class T, class F> void set_callback(F &&t);
-      explicit operator bool() const;
-      bool operator!() const;
+      explicit                         operator bool() const;
+      bool                             operator!() const;
```

Comment:
methods too?  this seems odd to me. I  like this for members but for  methods is a bit weird.

Reply:
Yeh, I don't see adoption for this specifically :/